packages/core/src/index.ts

```diff
@@ -47,6 +47,7 @@ export class LitElement extends BasicElement {
  */
 export { WarningNotice } from './notices/WarningNotice.js';
 export { DeprecationNotice } from './notices/DeprecationNotice.js';
+export { ErrorNotice } from './notices/ErrorNotice.js';
```

Comment:
This sounds like it works the same way with other Notice classes above. Let's come up with a new distinguished name.